testdata/intake-v2/transactions.ndjson

```diff
@@ -1,5 +1,5 @@
 {"metadata": {"service": {"name": "1234_service-12a3","node": {"configured_name": "node-123"},"version": "5.1.3","environment": "staging","language": {"name": "ecmascript","version": "8"},"runtime": {"name": "node","version": "8.0.0"},"framework": {"name": "Express","version": "1.2.3"},"agent": {"name": "elastic-node", "activation_method": "some_activation_method","version": "3.14.0"}},"user": {"id": "123user", "username": "bar", "email": "bar@user.com"}, "labels": {"tag0": null, "tag1": "one", "tag2": 2}, "process": {"pid": 1234,"ppid": 6789,"title": "node","argv": ["node","server.js"]},"system": {"hostname": "prod1.example.com","architecture": "x64","platform": "darwin", "container": {"id": "container-id"}, "kubernetes": {"namespace": "namespace1", "pod": {"uid": "pod-uid", "name": "pod-name"}, "node": {"name": "node-name"}}},"cloud":{"account":{"id":"account_id","name":"account_name"},"availability_zone":"cloud_availability_zone","instance":{"id":"instance_id","name":"instance_name"},"machine":{"type":"machine_type"},"project":{"id":"project_id","name":"project_name"},"provider":"cloud_provider","region":"cloud_region","service":{"name":"lambda"}}}}
-{"transaction": { "id": "945254c567a5417e", "trace_id": "0123456789abcdef0123456789abcdef", "parent_id": "abcdefabcdef01234567", "type": "request", "duration": 32.592981, "timestamp": 1496170407154000, "span_count": { "started": 43 }}}
+{"transaction": { "id": "945254c567a5417e", "trace_id": "0123456789abcdef0123456789abcdef", "parent_id": "abcdefabcdef01234567", "type": "request", "duration": 32.592981, "timestamp": 1496170407154000, "span_count": { "started": 43 }, "otel": { "attributes" : { "elastic.profiler_stack_trace_ids" : ["foo", "bar", "foo"]}}}}
```

Comment:
system test is failing due to this change, but confirmed with Jonas that it is expected.
```
=== RUN   TestIntake/Transactions
    intake_test.go:71:   []any{
                map[string]any{"@timestamp": []any{string("2017-05-30T18:53:27.154Z")}, "agent.activation_method": []any{string("some_activation_method")}, "agent.name": []any{string("elastic-node")}, "agent.version": []any{string("3.14.0")}, ...},
                map[string]any{
                        ... // 17 identical entries
                        "data_stream.namespace": []any{string("default")},
                        "data_stream.type":      []any{string("traces")},
        -               "event.outcome":         []any{string("unknown")},
        +               "event.outcome":         []any{string("success")},
        +               "event.success_count":   []any{float64(1)},
                        "host.architecture":     []any{string("x64")},
                        "host.hostname":         []any{string("node-name")},
                        ... // 28 identical entries
                        "service.version":                      []any{string("5.1.3")},
                        "span.id":                              []any{string("945254c567a5417e")},
        +               "span.kind":                            []any{string("SERVER")},
                        "timestamp.us":                         []any{float64(1.496170407154e+15)},
                        "trace.id":                             []any{string("0123456789abcdef0123456789abcdef")},
                        "transaction.duration.us":              []any{float64(32592)},
                        "transaction.id":                       []any{string("945254c567a5417e")},
        +               "transaction.profiler_stack_trace_ids": []any{string("foo"), string("bar"), string("foo")},
                        "transaction.representative_count":     []any{float64(1)},
        +               "transaction.result":                   []any{string("Success")},
                        "transaction.sampled":                  []any{bool(true)},
                        "transaction.span_count.started":       []any{float64(43)},
                        ... // 5 identical entries
                },
                map[string]any{"@timestamp": []any{string("2017-05-30T18:53:27.154Z")}, "agent.activation_method": []any{string("some_activation_method")}, "agent.name": []any{string("elastic-node")}, "agent.version": []any{string("3.14.0")}, ...},
                map[string]any{"@timestamp": []any{string("2018-07-30T18:53:42.281Z")}, "agent.activation_method": []any{string("some_activation_method")}, "agent.ephemeral_id": []any{string("justanid")}, "agent.name": []any{string("elastic-ruby")}, ...},
                map[string]any{"@timestamp": []any{string("2021-09-15T20:11:06.365Z")}, "agent.activation_method": []any{string("some_activation_method")}, "agent.name": []any{string("elastic-node")}, "agent.version": []any{string("3.14.0")}, ...},
          }
        
        Test failed. Run `make check-approvals` to verify the diff.
```

Reply:
updated approvals